src/python/psic/assigner/user_ref.py

```diff
@@ -0,0 +1,22 @@
+class User:
+
+    id: str  # The user's unique authentication ID
+    nickname: str  # The user's nickname
+    role: str  # The user's role (may be str or int in implementation)
+
+    def __init__(self, user_id: str, role: str = 'User', nickname: str = 'Unknown'):
+        self.id = user_id
+        self.nickname = nickname
+        self.role = role
+
+    def __str__(self):
+        return self.get_id()
+
+    def get_id(self):
+        return self.id
+
+    def get_nickname(self):
+        return self.nickname
+
+    def get_role(self):
+        return self.role
```

Comment:
This file is actually not needed, and may be removed in the future. Currently, the program just uses the user_id and doesn't use anything else, so there's no need to pass a whole object just for a user unless more information needs to be displayed on the dashboard to the user.